src/pubsub.rs

```diff
@@ -74,6 +74,13 @@ type WsSink = SinkWrite<
     >,
 >;
 
+#[derive(Clone)]
+pub struct Config {
+    pub ttl: Duration, // time to live
+    pub distance: u32, // slot distance
```

Comment:
slot_distance would be more clear naming

Reply:
Fixed, you can recheck